plugin/trino-resource-group-managers/src/main/resources/db/migration/mysql/V8__add_physical_data_scan_to_resource_groups.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE resource_groups ADD COLUMN soft_physical_data_scan_limit VARCHAR(128);
```

Comment:
Why varchar and not `integer`?

Reply:
The expected input for this property is a data size string e.g. `1MB` so it needs to be `varchar`. This is consistent with `softMemoryLimit` if absolute value is used.